includes/class-wc-stripe-status.php

```diff
@@ -194,4 +195,62 @@ public function render_status_report_section() {
 		</table>
 		<?php
 	}
+
+	/**
+	 * Add Stripe tools to the Woo debug tools.
+	 *
+	 * @param array $tools List of current available tools.
+	 */
+	public function debug_tools( $tools ) {
+		if ( WC_Stripe_Subscriptions_Helper::is_subscriptions_enabled() ) {
+			$tools['list_detached_subscriptions'] = [
```

Comment:
Nit: should we prefix this key to make collisions way less likely?
```suggestion
			$tools['wc_stripe_list_detached_subscriptions'] = [
```